templates/unstable-vite/app/root.tsx

```diff
@@ -15,12 +15,12 @@ export default function App() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <Meta />
         <Links />
+        <DevScripts />
       </head>
       <body>
         <Outlet />
         <ScrollRestoration />
         <Scripts />
-        <LiveReload />
```

Comment:
We probably need this change in `templates/spa/root.tsx` as well which is based on vite